scripts/coccinelle/tor-coccinelle.h

```diff
@@ -1,3 +1,21 @@
 #define MOCK_IMPL(a, b, c) a b c
```

Comment:
This header needs some documentation, and the text in the commit message is a good starting point:

Note that this header file behaves a bit strangely.  It is used by
coccinelle just for the purpose of knowing how to parse
difficult-to-parse stuff.  It doesn't need to produce good C -- just
grammatical C.